CONTRIBUTING.md

```diff
@@ -154,6 +154,7 @@ Make sure that [Travis](http://www.travis-ci.org) greenlights the pull request w
  - use lower case with underscores for method names
  - it is generally preferred to use ASCII operators and identifiers over
    Unicode equivalents whenever possible
+ - function signature should not be followed by an empty line
```

Comment:
or something like: `no empty line before end` in julia and `no empty line before }` in c?